lib/inspec/dist.rb

```diff
@@ -0,0 +1,9 @@
+module Inspec
+  module Dist
+    # When referencing a product directly, like InSpec
+    PRODUCT = 'InSpec'.freeze
+
+    # The inspec executable
+    EXEC = 'inspec'.freeze
```

Comment:
Since our minimum version of ruby is 2.4, I'd suggest using frozen string literals with the magic comment:

```ruby
# frozen_string_literal: true
```

and removing the manual freeze calls.

That said, I don't see this as a blocker at all.